api/middleware/middleware_test.go

```diff
@@ -42,8 +41,10 @@ func TestJwtMiddleware(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	logger := log.New()
-	logger.Out = os.Stdout
+	logger, err := log.NewLogger("stdout", true)
```

Comment:
`stdout` isn't a real filepath is it?

Reply:
```suggestion
	logger, err := log.NewLogger("", true)
```

Reply:
It's not a real file path, I think it's a reserved word by `za`p to log to stdout

Reply:
it's already the default

Reply:
handled here: https://github.com/RTradeLtd/Temporal/blob/f63e7311fe6f39ff904ae8574c1c53d6d232a169/log/log.go#L37

Reply:
see usage in orchestrator: https://github.com/RTradeLtd/ipfs-orchestrator/blob/master/daemon.go#L27 provides a logfile but still goes to stdout